shellSort/python/shell_sort.py

```diff
@@ -0,0 +1,22 @@
+def shellSort(arr):
+    gap = int((len(arr)/2))
+    while gap > 0:
+        for i in range (gap, len(arr)):
+            temp = arr[i]
+            j = i
+            while j >= gap and arr[j - gap] > temp:
+                arr[j] = arr[j-gap]
+                j -= gap
+            arr[j] = temp
+        gap/=2  
+        gap = int(gap)

```

Comment:
What does this function return?

Reply:
Please wait. I'll correct it within 15mins.